site-to-site-vpn/azuredeploy.json

```diff
@@ -0,0 +1,406 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": {
+            "type": "String",
+            "metadata": {
+                "description": "Region where the resources will be deployed"
+            },
+            "defaultValue": "Central US",
+            "allowedValues": [
+                "North Central US",
+                "East US 2",
+                "East Asia",
+                "West US",
+                "Central US"
+            ]
+        },
+        "vpnType": {
+            "type": "String",
+            "metadata": {
+                "description": "Route based or policy based"
+            },
+            "defaultValue": "RouteBased",
+            "allowedValues": [
+                "RouteBased",
+                "PolicyBased"
+            ]
+        },
+        "enableBgp": {
```

Comment:
Looks like there is only one allowed value which doesn't give the user much choice. Might be a better idea to just have this as a variable.